index.html

```diff
@@ -220,7 +220,23 @@ <h4 id="popup-title"></h4> <i class="far fa-window-close" id="closePopup"></i>
               <ul class="skills" id="popup-skills">
```

Comment:
- [ ] Please make sure that the logo is aligned more to the right to match the design.

- [ ] Please fix this weird space around your headline section which is caused by this useless margin also. 

- [ ] Please the text and social media icon links are in the center of the page but aligned to the left.

![image](https://user-images.githubusercontent.com/74744787/215268190-5d6c04b2-1d13-4d78-8322-f3ce5e98d1eb.png)

| Your design  |Figmas |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/215268227-881b7baa-74a0-4197-a4dc-97c47285ac92.png) | ![image](https://user-images.githubusercontent.com/74744787/215268252-07175522-815f-4d41-bba1-7e9a36358a17.png)  |